### PR TITLE
[s]Fix survival capsules (again)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Whatever new RANGE_TURFS thing is doesn't work, can be used to grief the station/bypass walls again

### CODE REVIEW

**Comment by xxalpha** [comment]
>grief the station/bypass walls again

please explain

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -188,7 +188,7 @@
 		used = TRUE
 		sleep(50)
 		var/clear = TRUE
-		for(var/unclear in RANGE_TURFS(3, src))
+		for(var/unclear in range(3,src))
 			var/turf/T = unclear
```
Comment (Giacom) [inline code/modules/mining/mine_items.dm]: Isn't this going to be invalid since range() will include non-turf objects?

Comment (KorPhaeron) [inline code/modules/mining/mine_items.dm]: Oh right yeah good point, I forgot I made the loop typeless. Thanks.

**Comment by Xhuis** [comment]
I never understood why shelter capsules worked how they did. It seems like it'd make more thematic sense and be more easy to accomplish in the backend if they just created a temporary sleeper-esque pod that you jump into, then manually recompress after you get out.

But I digress. :+1: for no griefing

**Comment by Supermichael777** [comment]
thanks kor. sorry about the textblock post but my phones browser editor sucks.

**Pull request merged**